Makefile

```
@@ -112,7 +112,7 @@ pip_dev_deps:
 
 venv_dev:
 	# Create a virtual environment for development.
-	virtualenv -p python2 venv_dev && \
+	virtualenv -p python2 venv_dev --prompt="(CodeChecker v-env [DEV]) " && \
```

I am thinking on the following names:
- development: `(cc-venv [dev]) `
- production: `(cc-venv) ` or `(cc-venv [prod]) `

What do you think @whisperity

---

Will it be obvious? `cc` is also the name for the C compiler. My intention was to make *CodeChecker* stand out.

---

What if we drop the v-env and just use `(CodeChecker)` and `(CodeChecker-dev)`. I prefer the short prompts too but `cc` could be misleading.

---

I like the idea about the prompt, however, that would also make-believe that `CodeChecker` (as the built thing) is available in the `PATH`. (Some tools install into venvs via pip by putting themselves an entrypoint to `venv/bin` so sourceing into the venv allows you to run, e.g. `rst2html`.) Maybe we should, in that case, modify the `Makefile` so the venv symlinks back into the build? Or the build is generated into the venv? It would also make it more user friendly so the user doesn't have to modify their PATH if they're running a self-built package.

---

This patch contains a merge conflict. Could you please resolve it. The problem will be that now we are creating the venv this way: `python3 -m venv venv_dev`. You can also pass the prompt option to this one.

Also for me now it looks like this:
```sh
((CodeChecker v-env [DEV]) ) csordasmarton@783566fb:~/codechecker$
```
It contains an extra space and to much parentheses. Maybe we can change it to this one:
```suggestion
	virtualenv -p python2 venv_dev --prompt="CodeChecker v-env [DEV]" && \
```
Until the patch is fixed I will merge this PR.